content/blog/2019/10/2019-10-30-speaker-blog-how-to-build-the-top-mobile-game.adoc

```diff
@@ -0,0 +1,161 @@
+---
+layout: post
+title: "Speaker blogpost: A Cloud Native Jenkins"
```

Comment:
I am not sure this is the intended post title

Reply:
We can remove "Speaker blogpost:"

Reply:
Ah, it is indeed not intended :D

Reply:
This should now be fixed